sockets/utils/getCurrentScriptSource.js

```diff
@@ -11,7 +11,8 @@ function getCurrentScriptSource() {
 
   // Fallback to getting all scripts running in the document.
   const scriptElements = document.scripts || [];
-  const currentScript = scriptElements[scriptElements.length - 1];
+  const nonNullScriptElements = [...scriptElements].filter((elm) => elm.getAttribute('src'));
+  const currentScript = nonNullScriptElements[nonNullScriptElements.length - 1];
   if (currentScript) {
     return currentScript.getAttribute('src');
   }
```

Comment:
I think here we can just check if the length of the filtered elements is non-zero and return the `src`.

Reply:
Yeah, I think it would be good to check if the length is non-zero.